### PR TITLE
Fix tasks filters

### LINKED ISSUE #1350: On task assignee single select picker, the current option is not ticked
On Tasks Page, we are using a SingleEntitySelect to pick the task assignee (through UserPicker). However, the current assignee is displayed as selected with a tick.

image

However, selected tick behavior should have been implemented in MenuItemSelectAvatar.

This ticket is about investigating this issue and fixing it! Should be a "small" issue!

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Fixes #1350

### CODE REVIEW

**Review by gitstart-twenty — COMMENTED** [review]
⚠️ Initially there's no `selectedEntity` (it is `undefined` when passed into `SingleEntitySelectBase`). The value is only set when you click on an item. Here, I check if there's a `filter` selected, but no `selectedEntity`, and then set the `selectedEntity` to the value of the `filter`.

Still thinking on getting a better solution🤔

**Review by lucasbordeau — COMMENTED** [review]
Almost ! 

Hint : use `filterCurrentlyEdited`

File: front/src/modules/ui/view-bar/components/FilterDropdownEntitySearchSelect.tsx [hunk 1]
```diff
@@ -36,6 +37,11 @@ export const FilterDropdownEntitySearchSelect = ({
     ViewBarRecoilScopeContext,
   );
 
+  const [filters] = useRecoilScopedState(
```
Comment (charlesBochet) [inline front/src/modules/ui/view-bar/components/FilterDropdownEntitySearchSelect.tsx:40]: let's not import filters here and only use useFilterCurrentlyEdited in the code

File: front/src/modules/ui/view-bar/components/FilterDropdownEntitySearchSelect.tsx [hunk 2]
```diff
@@ -75,8 +81,15 @@ export const FilterDropdownEntitySearchSelect = ({
   useEffect(() => {
     if (!filterCurrentlyEdited) {
       setFilterDropdownSelectedEntityId(null);
+    } else if (!entitiesForSelect.selectedEntities[0] && filters[0]) {
+      setFilterDropdownSelectedEntityId(filters[0].value);
```
Comment (charlesBochet) [inline front/src/modules/ui/view-bar/components/FilterDropdownEntitySearchSelect.tsx:85]: setFilterDropdownSelectedEntityId(filterCurrentlyEdited.value)

**Review by charlesBochet — APPROVED** [review]
LGTM, thanks!

**Pull request merged**